app/models/messages/Consignment.scala

```diff
@@ -57,7 +57,7 @@ object Consignment {
       (__ \ "modeOfTransportAtTheBorder").readNullable[String] and
       (__ \ "TransportEquipment").readNullable[List[TransportEquipment]] and
       (__ \ "LocationOfGoods").readNullable[LocationOfGoods] and
-      (__ \ "DepartureTransportMeans").readWithDefault[List[DepartureTransportMeans]](Nil).map(_.headOption) and
+      (__ \ "DepartureTransportMeans").readNullable[Seq[DepartureTransportMeans]] and
```

Comment:
I have a feeling the reads and writes can be replaced with the bog standard `implicit val format: Format[Consignment] = Json.format[Consignment]` now

Reply:
Yep this works